src/restc.erl

```diff
@@ -59,7 +59,7 @@
 -type content_type() :: multi | json | xml | percent | png.
 -type body()         :: binary()             |
                         jsx:json_term()      |
-                        erlsom:simple_form() |
+                        term() | % check erlsom:simple_form/1,2
```

Comment:
I don't know of a better way to do this.

1. the `erlsom:simple_form()` type doesn't exist
2. in our tests, for example, the return is a tuple, but without any _proper_ "form"

My goal in adding the comment is to have the consumer look at the dependency, if they need it for that specific type.